packages/web-forms/src/lib/isMobileDevice.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Regular expression patterns for detecting mobile devices.
+ */
+const SMARTPHONE_PATTERNS: RegExp[] = [
```

Comment:
Ideally we could avoid User-Agent regex and directly check whether camera input is available. It looks like this is possible by checking the `accept` attribute value! https://stackoverflow.com/a/51273905/137744

Reply:
It works in IPhone and Android, thanks!